lib/commons/table/is-data-cell.js

```diff
@@ -13,8 +13,8 @@ function isDataCell(cell) {
   if (!cell.children.length && !cell.textContent.trim()) {
     return false;
   }
-  const role = cell.getAttribute('role');
-  if (isValidRole(role)) {
+  const role = getExplicitRole(cell);
+  if (role) {
```

Comment:
`getExplicitRole` already validates the role if it returns, otherwise it returns `null` for an invalid role